ecosystem/sep-0031.md

```diff
@@ -504,6 +543,34 @@ Example response:
 }
 ```
 
+```json
+{
+  "transaction": {
+      "id": "82fhs729f63dh0v4",
+      "status": "completed",
+      "amount_in": "110",
+      "amount_out": "95",
+      "amount_fee": "5",
+      "started_at": "2017-03-20T17:05:32Z",
+      "stellar_transaction_id": "b9d0b2292c4e09e8eb22d036171491e87b8d2086bf8b265874c8d182cb9c9020",
+      "stellar_account_id": "GBRPYHIL2CI3FNQ4BXLFMNDLFJUNPU2HY3ZMFSHONUCEOASW7QC7OX2H",
+      "stellar_memo": "123456789",
+      "stellar_memo_type": "id",
+      "refunds": {
+        "amount_refunded": "10",
+        "amount_fee": "0",
+        "payments": [
```

Comment:
Can we add an example of "id_type": "external"?

Reply:
There is no `id_type` attribute (and no `external` attribute value) for SEP-31 transactions, since the receiving anchor always receives payments via Stellar.